Slack.Automation/Promact.Core.Repository/SlackUserRepository/SlackUserRepository.cs

```
@@ -29,6 +29,40 @@ public void AddSlackUser(SlackUserDetails slackUserDetails)
         }
 
         /// <summary>
+        /// Method to update slack user 
+        /// </summary>
+        /// <param name="slackUserDetails"></param>
+        public void UpdateSlackUser(SlackUserDetails slackUserDetails)
+        {
+            var user = _slackUserDetails.FirstOrDefault(x => x.UserId == slackUserDetails.UserId);
```

What happens if User is not found? How will you provide feedback to end user?

---

We can throw an exception